ddl/partition.go

```diff
@@ -1314,34 +1301,19 @@ func (w *worker) onExchangeTablePartition(d *ddlCtx, t *meta.Meta, job *model.Jo
 	}
 
 	// both pt and nt set the maximum auto_id between ntBaseID and ptBaseID
-	if ntBaseID > ptBaseID {
```

Comment:
The goal of these lines is to set the auto IDs to the maximum between the **normal** table and the **partitioned** table.

Comparing with this PR, the original logic can reduce 1 write because the `ptBaseID` is already the maximum one. But IMO I think it is unnecessary because it also decreases the readability.